### ISSUE TITLE
Standardize which language the GLSL lsps get filtered by 

### ISSUE THREAD

**Issue by Alextibtab** [mason-org/mason-registry#6218]
I was recently doing some graphics programming and initially when I tried to install an LSP for GLSL I only saw glslls in Mason. After looking for alternatives I found glsl_analyzer which I then locally installed and was wondering why it wasn't included in mason but after checking I realised that it was just not when I filtered by GLSL. 

I'm not sure which language would be more appropriate GLSL or OpenGL but I think it would cause less confusion if both were set to the same language. 

<img width="1921" alt="image" src="https://github.com/mason-org/mason-registry/assets/48660540/c4d548b5-500a-42a9-8fc4-6c7a88678830">

<img width="1921" alt="image" src="https://github.com/mason-org/mason-registry/assets/48660540/e250b002-ffa7-4944-a874-c617c04ef03e">

**Comment by Conarius** [comment]
Yeah, it's because they have different languages in the field of the `package.yaml`. I can understand why it's confusing so one language or both would be great but it's ultimately up to the maintainer
